katharsis-ui/pom.xml

```diff
@@ -9,18 +9,99 @@
     </parent>
 
     <artifactId>katharsis-ui</artifactId>
-    <packaging>jar</packaging>
+  	<packaging>war</packaging>
```

Comment:
This isn't what we want if we want maximum portability. Used in this way, it would have to be in it's own app space which may or may not make sense. It should be a jar that can be pulled in as a dependency into a war on a case-by-case basis.

Also, since this is a JS package, it would be nice if we also had an NPM repo we could deploy too